app/main.py

```diff
@@ -1,2 +1,73 @@
+from typing import Any, Hashable
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Node:
+    key: Hashable
+    hash_val: int
+    value: Any
+
+
+@dataclass
 class Dictionary:
-    pass
+    initial_capacity = 8
+    load_factor = 2 / 3
+    size = 0
+    hash_table: list[list[Node]] = field(init=False)
+
+    def __post_init__(self) -> None:
+        self.hash_table = [None] * self.initial_capacity
+
+    def line_stop(self) -> int:
+        return int(self.initial_capacity * self.load_factor)
+
+    def get_index(self, key: Hashable) -> int:
+        return hash(key) % self.initial_capacity
+
+    @staticmethod
+    def get_hash(key: Hashable) -> int:
+        return hash(key)
```

Comment:
```suggestion
    def _line_stop(self) -> int:
        return int(self.initial_capacity * self.load_factor)

    def _get_index(self, key: Hashable) -> int:
        return hash(key) % self.initial_capacity

    @staticmethod
    def _get_hash(key: Hashable) -> int:
        return hash(key)
```

I also think it's good practice to use private attributes here